src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/service/HomeOfficeSearchService.java

```diff
@@ -31,8 +34,19 @@ public HomeOfficeSearchService(
     public HomeOfficeSearchResponse getCaseStatus(String homeOfficeReferenceNumber) {
 
         HomeOfficeSearch request = makeRequestBody(homeOfficeReferenceNumber);
+        ObjectWriter ow = new ObjectMapper().writer().withDefaultPrettyPrinter();
+        try {
+            log.info("HomeOffice-CaseSearch request: {}", ow.writeValueAsString(request));
+        } catch (JsonProcessingException e) {
+            e.printStackTrace();
```

Comment:
shouldn't eat up the Exception here.  Just re-throw it so we can see it in logs and in appinsights.

Reply:
Not sure it needs a try/catch block?

Reply:
Throw exception and removed exception handling